### PR TITLE
Update README.md with changes related to LLava-next multi card support

### PR DESCRIPTION
**Pull request by tthakkal**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by mandy-li** [comment]
@tthakkal , can we add llava-next  multi-card example command to README as well?

**Comment by tthakkal** [comment]
> @tthakkal , can we add llava-next multi-card example command to README as well?

@mandy-li Added

**Review by mandy-li — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -256,6 +254,8 @@ docker run -p 8080:80 \
    -v $volume:/data \
    -e HABANA_VISIBLE_DEVICES=all \
    -e OMPI_MCA_btl_vader_single_copy_mechanism=none \
+   -e TEXT_GENERATION_SERVER_IGNORE_EOS_TOKEN=true \
+   -e PT_HPU_ENABLE_LAZY_COLLECTIVES=true \
```
Comment (regisss) [inline README.md:258]: We need to set `PT_HPU_ENABLE_LAZY_COLLECTIVES` to True for single card usage?

**Review by regisss — APPROVED** [review]
I left one comment but it's not a blocker.

**Pull request merged**